.idea/externalDependencies.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<project version="4">
+  <component name="ExternalDependencies">
+    <plugin id="Git4Idea" />
+    <plugin id="androidx.compose.plugins.idea" />
+    <plugin id="com.facebook.ktfmt_idea_plugin" />
+    <plugin id="com.faendir.intellij.gradle-version-catalogs-plugin" />
+    <plugin id="com.intellij.gradle" />
+    <plugin id="com.yoheimuta.intellij.plugin.protolint" />
+    <plugin id="idea.plugin.protoeditor" />
+    <plugin id="org.intellij.plugins.markdown" />
+    <plugin id="org.jetbrains.android" />
+    <plugin id="org.jetbrains.kotlin" />
+    <plugin id="org.jetbrains.plugins.github" />
+    <plugin id="org.jetbrains.plugins.yaml" />
+    <plugin id="org.toml.lang" />
```

Comment:
Ok now that I don't have conflicts, I tried this out and have a red popup in Android Studio that "Required plugins have not been loaded". In VS Code at least it was just "recommended", can we do that here? I was able to work perfectly fine without these plugins.

Reply:
Doesn't look like I can set them to "Recommended", unfortunately. I removed the ones that aren't necessary like GitHub, but I think I'll leave in the formatting ones like `toml` and `yaml`. This is so we can make sure we have our formatting correct, after all.